common-rendering/src/components/keyEvents.tsx

```diff
@@ -75,8 +75,16 @@ const keyEventWrapperStyles = (
 	`}
 `;
 
-const listStyles = (supportsDarkMode: boolean): SerializedStyles => css`
-	li::before {
+const linkStyles = (
+	supportsDarkMode: boolean
+): SerializedStyles => css`
+	text-decoration: none;
+
+	&:hover:before {
+		background-color: ${neutral[0]};
+	}
+
+	&:before {
```

Comment:
I know this technically works, but the CSS3 syntax is the double-double-colon `::` for pseudo-elements, to distinguish them from states.

https://developer.mozilla.org/en-US/docs/Web/CSS/::before#syntax

Reply:
Maybe we need a lint rule and it’s actually outside the scope of this PR?